docs/api/mount/hasAttribute.md

```diff
@@ -1,17 +1,14 @@
-# hasAttribute(attribute, value)
```

Comment:
I agree, but people will need to change tests that rely on hasAttribute. 

We could just deprecate hasAttribute (remove it from docs, add a warning that it will be removed in later versions, but keep the same functionality)